core/src/protobuf.py

```diff
@@ -207,6 +210,9 @@ def load_message(
         fields = msg_type.get_fields()
         field_cache[msg_type] = fields
 
+    if getattr(msg_type, "UNSTABLE", False) and not experimental_enabled:
```

Comment:
please also put `UNSTABLE = False` on `MessageType` so that we don't have to use getattr this way

Reply:
Fixed: 436bfea27dd757cbeb067750d081dd92fb7a2805.